[2025-08-02T22:07:21.448+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-02T22:07:21.477+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_process_all_files_v1.process_json_file manual__2025-08-02T22:07:15.539522+00:00 [queued]>
[2025-08-02T22:07:21.490+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_process_all_files_v1.process_json_file manual__2025-08-02T22:07:15.539522+00:00 [queued]>
[2025-08-02T22:07:21.492+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-02T22:07:21.512+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_json_file> on 2025-08-02 22:07:15.539522+00:00
[2025-08-02T22:07:21.530+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1186) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-02T22:07:21.533+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'gcs_process_all_files_v1', 'process_json_file', 'manual__2025-08-02T22:07:15.539522+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/gcs_load_dag v2.py', '--cfg-path', '/tmp/tmp22jnn6j3']
[2025-08-02T22:07:21.535+0000] {standard_task_runner.py:63} INFO - Started process 1194 to run task
[2025-08-02T22:07:21.535+0000] {standard_task_runner.py:91} INFO - Job 56: Subtask process_json_file
[2025-08-02T22:07:21.638+0000] {task_command.py:426} INFO - Running <TaskInstance: gcs_process_all_files_v1.process_json_file manual__2025-08-02T22:07:15.539522+00:00 [running]> on host 06d424bf0ee1
[2025-08-02T22:07:21.784+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_process_all_files_v1' AIRFLOW_CTX_TASK_ID='process_json_file' AIRFLOW_CTX_EXECUTION_DATE='2025-08-02T22:07:15.539522+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-02T22:07:15.539522+00:00'
[2025-08-02T22:07:21.786+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-02T22:07:21.803+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-08-02T22:07:21.812+0000] {base.py:84} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-08-02T22:07:23.383+0000] {gcs.py:574} INFO - Data stream uploaded to raw/mcc_codes.csv in dataverse360-raw-data-rmk bucket
[2025-08-02T22:07:23.384+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-02T22:07:23.389+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-02T22:07:23.408+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=gcs_process_all_files_v1, task_id=process_json_file, run_id=manual__2025-08-02T22:07:15.539522+00:00, execution_date=20250802T220715, start_date=20250802T220721, end_date=20250802T220723
[2025-08-02T22:07:23.477+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-02T22:07:23.509+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-02T22:07:23.512+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
